sunpy/version.py

```diff
@@ -0,0 +1,5 @@
+from pkg_resources import get_distribution, DistributionNotFound
```

Comment:
This should probably have a comment stating
`# This file is for compatibility with astropy_helpers`